src/components/ui/hook-form/datepicker/index.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from '@emotion/styled';
+import {
+    DatePicker as DatePickerAntD,
+    DatePickerProps,
+    Flex,
+    Typography,
+} from 'antd';
+import { useController, UseControllerProps } from 'react-hook-form';
+import { BREAKPOINTS } from '~constants/breakpoints';
+
+type FormDatePickerBaseProps = {
+    className?: string;
+    label?: string;
+} & DatePickerProps &
+    UseControllerProps;
+
+export function FormDatePickerBase({
+    name,
+    label,
+    className,
+    rules,
+    defaultValue = '',
+    id,
+    control,
+    ...props
+}: FormDatePickerBaseProps) {
+    const { Text } = Typography;
+    const { field, fieldState } = useController({
```

Comment:
вместо пропса control
```suggestion
    const { control } = useFormContext();
    const { field, fieldState } = useController({
```